ros2topic/ros2topic/verb/pub.py

```diff
@@ -99,6 +99,9 @@ def set_msg_fields(msg, values):
                 raise SetFieldError(
                     '{field_name}.{e.field_name}'.format_map(locals()),
                     e.exception)
+        except ValueError as e:
+            raise SetFieldError(
+                '{field_name}'.format_map(locals()), e)
```

Comment:
No, not with `format_map`. There is already plenty of other code using this snippet.